source/content/guides/secrets/01-introduction.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Pantheon Secrets Guide
+subtitle: Introduction
+description: Securely store secrets in the Pantheon Platform.
+contributors: [stovak]
+contenttype: [guide]
+innav: [true]
+categories: [secrets]
+cms: [drupal, wordpress]
+audience: [development]
+product: [secrets]
+integration: [--]
+tags: [reference, cli, local, terminus, workflow]
+permalink: docs/guides/secrets
+reviewed: "2024-07-30"
+showtoc: true
+---
+
+Pantheon Secrets is key to maintaining industry best practices for secure builds and application implementation. This feature provides a convenient mechanism for you to manage your secrets and API keys directly on the Pantheon platform.
+
+This guide covers features and use cases of the Pantheon Secrets feature; it could also be referred as Secrets Manager because that is the Terminus plugin name.
+
+## Features
```

Comment:
The feature list here should be aligned with what's specified in the plugin's README file